CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - PR #332 fix directed_hausdorff_distance's space_offsets name + documentation
 
 - PR #331 Use simplified `rmm::exec_policy`
+- PR #339 Pin gdal to 3.1.x
```

Comment:
just a reminder that manual changelog entries are no longer necessary for RAPIDS libraries. feel free to remove this or keep it. It will be overwritten with an automated changelog during release time.